tests/ballerina-unit-test/src/test/resources/test-src/connectors/sql/sql_actions_test.bal

```diff
@@ -1555,3 +1555,34 @@ function testCloseConnectionPool(string jdbcUrl, string userName, string passwor
     testDB.stop();
     return count;
 }
+
+function testReInitEndpoint(string jdbcUrl, string userName, string password) returns int {
+    endpoint jdbc:Client testDB {
+        url: jdbcUrl,
+        username: userName,
+        password: password,
+        poolOptions: { maximumPoolSize: 1 }
+    };
+
+    jdbc:ClientEndpointConfiguration config = {
+        url: jdbcUrl,
+        username: userName,
+        password: password,
+        poolOptions: { maximumPoolSize: 1 }
+    };
+
+    testDB.stop();
+
+    testDB.init(config);
```

Comment:
Same here. Lets try to change this test case a bit. Lets init with different config values (for example - different jdbcUrl , db name) and then run the test case. This way we make sure that a new config is used with init.